pkg/shared/util/version_mapping.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2022 The Numaproj Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"github.com/numaproj/numaflow-go/pkg/info"
+)
+
+type sdkConstraints map[info.Language]string
+
+var minimumSupportedSDKVersions = sdkConstraints{
+	info.Go:     "0.7.0-rc2",
```

Comment:
Quick question: is `0.7.0` > `0.7.0-rc2`?

Reply:
yes it is